### ISSUE TITLE
Question about the preprocessing script bam_cov.py 

### ISSUE THREAD

**Issue by Wenhui-bio** [calico/basenji#195]
Hi, It seems that the bam_cov.py script that converts bam file to bigwig file does not involve sequencing depth normalization. I need to compare the difference in predicted results between samples, but the predicted difference is consistent with the difference in sequencing depth!

**Comment by davek44** [comment]
Correct, we do not perform depth normalization in that script. I generally perform normalization afterward on the model outputs.
